src/store/juju/slice.ts

```diff
@@ -128,9 +129,20 @@ const slice = createSlice({
         state.modelData[modelInfo.uuid].info = modelInfo;
       }
     },
+    updateModelsError: (
```

Comment:
It looks like this needs a reducer test.

Reply:
Nice catch! Added the test.